src/components/CustomChannelComponent.tsx

```diff
@@ -176,13 +184,21 @@ export function CustomChannelComponent() {
           20,
   });
 
+  useDisableInputUntilReply({
+    lastMessage,
+    botUser,
+    currentUserId,
+    setIsMessageInputDisabled,
+  });
```

Comment:
let move `useState` into `useDisableInputUntilReply` hook
```suggestion
const { isMessageInputDisabled } = useDisableInputUntilReply({
    lastMessage,
    botUser,
    currentUserId,
  });
```

Reply:
oh nice, its done!